indico/modules/auth/client/js/signup.jsx

```diff
@@ -223,6 +283,34 @@ function Signup({
               />
             </Fieldset>
           )}
+          {termsRequireAccept && !!tos && !tosUrl && (
+            <>
+              <h3>{Translate.string('Terms of service')}</h3>
```

Comment:
This h3 doesn't look great... maybe a fieldset for the whole terms/privacy section would be better?

Reply:
Also I just noticed that this should use `<Translate>...</Translate>` since `Translate.string()` should only be used in places where you need a plain string